_studio/shared/umc/codec/h265_dec/include/umc_h265_frame.h

```diff
@@ -82,7 +82,9 @@ class H265DecoderRefPicList
 };
 
 // HEVC decoder frame class
-class H265DecoderFrame : public H265DecYUVBufferPadded, public RefCounter
+class H265DecoderFrame;
+typedef std::shared_ptr<std::shared_ptr<H265DecoderFrame>> pH265DecoderFrame;
```

Comment:
1. `pH265DecoderFrame` looks ugly. I mix it with variable declaration. Probably `H265DecoderFrameP` would be better.
2. `std::shared_ptr<std::shared_ptr>` - do you really need such construction and why?